### ISSUE TITLE
Create new go refs format and generate GO.references

### ISSUE THREAD

**Issue by alexsign** [geneontology/go-site#993]
Hi, 
I cannot download following any more.
http://www.geneontology.org/doc/GO.references
It gives 404 error. If it's planned removal, can you please point me to an alternative resource.

**Comment by kltm** [comment]
Duplicate of geneontology/helpdesk#180

**Comment by kltm** [comment]
We used to have a file http://52.27.86.54/doc/GO.references that no longer seems to have an analog. We now produce http://snapshot.geneontology.org/metadata/gorefs/, from the metadata at https://github.com/geneontology/go-site/tree/master/metadata/gorefs . 

We want to:

- [ ] create a new metadata metadata/go-refs.json, compiled from metadata/gorefs/goref-*.yaml
- [ ] convert this file into metadata/GO.references, following to previous format as much as possible
- [ ] add a 302 (temp) forward from /docs/GO.references to /metadata/GO.references

**Comment by kltm** [comment]
http://geneontology.org/doc/GO.references now temporarily working again.

**Comment by alexsign** [comment]
thanks @kltm Can I assume http://snapshot.geneontology.org/metadata/gorefs/ is new permanent location and all gorefs combined file will not be produced in the future?

**Comment by kltm** [comment]
@alexsign 

From https://github.com/geneontology/go-site/issues/993#issuecomment-462557136 , we will be producing a work-a-like for GO.references from a new format that we'll be producing (at a new location). In the interim, we've manually added the old file.
In general, we'd encourage people to use the versioned monthly releases, with the most recent at current.geneontology.org. There are special use cases where people may be interested in snapshot releases as well.

**Comment by cmungall** [comment]
Minor: would be nice to set mime type so it resolves as a text file in a browser rather than defaulting to download

**Comment by kltm** [comment]
Downgraded as temp in place.

**Comment by kltm** [comment]
Temporarily blocked by https://github.com/dougli1sqrd/yamldown/issues/3

**Comment by kltm** [comment]
Final steps are

- [ ] add script to pipeline
- [ ] check that output and map are okay

**Comment by kltm** [comment]
Changes pulled through to snapshot and release. All points hit, now waiting to clear.
